Library/Formula/tippecanoe.rb

```diff
@@ -23,8 +23,8 @@ def install
     path.write <<-EOS.undent
       {"type":"Feature","properties":{},"geometry":{"type":"Point","coordinates":[0,0]}}
     EOS
-    output = `#{bin}/tippecanoe -o test.mbtiles #{path}`.strip
-    assert_equal 0, $?.exitstatus
-    assert_equal "using layer 0 name test", output
+
+    assert system("#{bin}/tippecanoe -o #{testpath}/test.mbtiles #{path}")
+    assert File.exist?(File.join(testpath, "test.mbtiles"))
```

Comment:
@mikemcquaid thanks for the tips, it's good to become aware of Homebrew's own commands, like `safe_system`, for this type of stuff. Thanks again for the help.

Updated and pushed.